server_config.cc

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
```

Comment:
The convention we've had in our other files for includes is: 

* First include our custom `"headers.h"`
* Newline
* Then include standard library `<headers>`

For both groups of headers, we list the includes alphabetically.